src/client/components/ShoppingCart/RemoveFromShoppingCart.js

```diff
@@ -0,0 +1,83 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import './ShoppingCart.styles.css';
+
+function RemoveFromShoppingCart(props) {
+  const { product, shoppingCart, setShoppingCart } = props;
+
+  const removeProductFromShoppingCart = () => {
+    
+    setShoppingCart((prev) => prev.filter((item) => item.id !== product.id));
+    const result = shoppingCart.find(({ id }) => id === product.id);
+    if (result === undefined) {
+      // console.log('product does not exist in cart');
+    } else {
+      // console.log('remove product from the cart');
+      const index = shoppingCart.indexOf(result);
+      if (index > -1) {
+        shoppingCart.splice(index, 1);
+        
+      }
+    }
```

Comment:
Turn the if clause around and get rid of console statements:

```suggestion
if (result !== undefined) {
  const index = shoppingCart.indexOf(result);
  if (index > -1) {
    shoppingCart.splice(index, 1);      
  }
}
```

Reply:
done